refactor: Remove Bloop4j as it seems not to be used at all

tgodzik opened the pull request.

ckipp01 reviewed (changes requested):
Can we also remove the project definition in the [`build.sbt`](https://github.com/scalacenter/bloop/blob/bf9ef5b93198aa637ebdebe86241f40e9d4653b7/build.sbt#L311-L322).

tgodzik commented:
> Can we also remove the project definition in the [`build.sbt`](https://github.com/scalacenter/bloop/blob/bf9ef5b93198aa637ebdebe86241f40e9d4653b7/build.sbt#L311-L322).

Ach! I removed it from build.sbt in the other PR :man_facepalming: Fixed now

ckipp01 reviewed (approved):
LGTM!
